staging/src/k8s.io/kube-aggregator/go.mod

```diff
@@ -0,0 +1,28 @@
+module k8s.io/kube-aggregator
+
+require (
+	k8s.io/api v1.12.0
```

Comment:
could we use a version string which never changes here, like "master" ? What do other projects do?

Reply:
Was thinking about changing it to a real version when branching off for releases, instead of having the version already on master for the next release.